src/baseevents.cpp

```diff
@@ -74,6 +74,7 @@ bool BaseEvents::loadFromXml()
 
 		if (!success || !registerEvent(event, node)) {
 			delete event;
+			return false;
```

Comment:
The point is that the commit needs a boolean value return that indicates whether the reload was successful or not, if the reload fails it should return false not true (aka it shouldn't say everything is OK when it isen't).
And the same with this issue, the old system did not require a boolean value for a return value, but this one does to check if monsters / spells should be loaded aswell.

This is a thing I forgot when I created the pr and not checking every return :P